database/migrations/2020_01_01_135000_create_business_category_place_table.php

```diff
@@ -12,7 +12,7 @@ public function up()
     {
         Schema::create('business_category_place', function (Blueprint $table) {
             $table->id();
-            $table->foreignIdFor(app('business_category'));
```

Comment:
Let's add this class to the tipoff/support config. Can you do a PR for it?

Reply:
will do